workers/python/python/database.py

```diff
@@ -24,6 +25,8 @@
     cert.write(config.mongodb_ca_cert)
     cert.close()
 
+PymongoInstrumentor().instrument()
```

Comment:
Are you sure this works with the async version? Are the data from mongo collected?

If not we could try this, which explicitly mentions the support:
https://logfire.pydantic.dev/docs/integrations/pymongo/

Reply:
Yes, it seems to work just fine 👍